### PR TITLE
PJ_SIM_SUNBNB_2023-15 [FE] Listing Item Component

### PR DESCRIPTION
**Pull request by LennardUy104**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-15

# Definition of Done
* [x] Create Card component for all listings showing
    * [x] Picture of home
    * [x] Listing name
    * [x] Price - type - capacity
    * [x] Average Rating
    * [x] Number of Reviews

# Notes
Insert the component in the all component for checking

# Expected Output
image

# Screenshots/Recordings
image

### CODE REVIEW

File: fe/src/utilities/dummy/ListingData.json
Comment (andreenriquez11) [inline fe/src/utilities/dummy/ListingData.json:1]: Can you add varieties to the list especially the name, image, type for a better view. Thanks

**Review by andreenriquez11 — CHANGES_REQUESTED** [review]

File: fe/src/utilities/dummy/ListingData.json [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
...XfuO6seaQWFgBbQ24g8N/Cq7Yok6/91aj2ZzkqLeE2NG7BTIyknQ9G178SSLd9HuL5PyvguYSJme1swKWIL5uDk+qgCGb18KKtj4vGsnvXOsq9EFVZhpbTcd1Vu3kzow5OQ2Mv8DJ4I/8Ato95P7NeLZs0Do6yNBOgBR9WfOFG7qIqmmM2kNyz/wCiD7YzVpNsbUHxZv8A46f8dN28sKHcnOQGBaGL3RETIR75eSRdbngGAHCjnBbDwUMLQRoojdXVlzMbiQWe5JJ1GlBabd2p8iX/AOOP+Opl27tT/Dl/0B/sp/bCjcl5G7JO+BO8GW/qNcrHIjEFzdHyXNssTnS+m8CjobY2qfiSDvijH2kpNjNqb2ZV+d7kHtFJ30xUYzck5Hw6wiKVWW1mMVhcHXTMN4vxpuL5CTSRxKtlaNSpLKAGG8fGNvXWv7ux3xsZAv8A5YL/AKq1G2LxB87aEY7mkP2Y6lNLsdmFg/J3iVPnIL77E38OqiXafI2TEGNnzLkijh6L7xHpmPQOpufQKoPIx87aD/orMfaVqrIkJ8/FTSf+L/dLQ9SPsLRdbydqvnSkfOcD92KsryZhSBoGxEPNtIspu1zmVcosRIthbhasXmsINwnP+kv3NXmfCj+6lPfMPujqXqR9k3Eu/wBFdnjzsRF4Zv8AlNSpgdmxxvF7oBjkKM6iNjcxkldcp3Emspp8PwwwPfNL+yy004yMebhofHnW+1IRR/bEVxLDpsuI54jKZEIKEKoswsVOsY3H2UKbXxERBeMOJCSTcR2JJub2HfwrRx+0tCObiW4+LDH+F/XesvDwsSXuBcEWCgWF7iwtpQprIrKmy8e0ZLBc5ZcrDVba71ykDSw333VDtbGmTKoSxVmIPxjm1NzuOuulWsXimO9ibXAv2m59dZefpkngPbWkXe46OzeQtSMJiL7+f/dJXT65n5EDfC4j6f8AdR10ytlgYqVKlTA8oY5dfBQ//kR+xqVKonxA4zL8EO9/bHWFB5/in268pVzxLK0+8/OqNq9pVsiWTYfdRVsTzW7/ALhSpVEgZupup4pUqzYhjVA1KlUsCtJTBvpUqlkky0jSpVkxDa8NKlQIa1NNKlSQDaYaVKqQGXjt4+cPbViPd/PXSpVr0MxJ97fOPtqi29vD769pV0QwWdp8hf8AZMR9P+6Sun0qVbIBUqVKmB//2Q==",
```
Comment (andreenriquez11) [inline fe/src/utilities/dummy/ListingData.json]: why is this so long?

File: fe/src/components/ListingsList.vue
Comment (andreenriquez11) [inline fe/src/components/ListingsList.vue:1]: Remove extra spaces on the bottom

**Review by andreenriquez11 — CHANGES_REQUESTED** [review]

**Comment by LennardUy104** [comment]
@andreenriquez11 Resolve Changes already

**Review by andreenriquez11 — APPROVED** [review]
looks good to me, thanks

File: fe/src/components/ListingCard.vue [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-
-const listing = ref({});
-
-onMounted(() => {
-  listing.value = listingData;
-});
-
+const  props  = defineProps(['homeItem']);
+const { homeItem } = toRefs(props);
+console.log(homeItem);
```
Comment (roseaugusto) [inline fe/src/components/ListingCard.vue]: remove console

**Review by roseaugusto — CHANGES_REQUESTED** [review]

**Pull request merged**